src/components/BookList.js

```diff
@@ -9,7 +9,7 @@ const BookList = ({ Books, title, handleDelete }) => (
         <h2>{book.title}</h2>
         <p>{book.author}</p>
         <div className="delete">
-          <button type="button" onClick={() => handleDelete(book.id)}>
+          <button type="button">
```

Comment:
- [ ]  Remove book functionality is not working I would recommend fixing your functionality by importing removeBook function from your newly created book reducer